tox.ini

```diff
@@ -65,6 +65,7 @@ setenv =
     dev: VIA_URL = {env:VIA_URL:http://localhost:9083}
     dev: SENTRY_ENVIRONMENT = {env:SENTRY_ENVIRONMENT:dev}
     dev: SESSION_COOKIE_SECRET = {env:SESSION_COOKIE_SECRET:notasecret}
+    dev: WEB_CONCURRENCY = {env:WEB_CONCURRENCY:2}
```

Comment:
This might be worth adding a comment to, as normally you grep to find out what things are, but grepping for this won't get you anything in our code, which could be confusing.